### PR TITLE
Document the startup procedure

### PR DESCRIPTION
**Pull request by phillipj**
This documents the most basic "setup" needed to get the bot up and running.

### CODE REVIEW

**Comment by phillipj** [comment]
Hm... Just realized the current procedure of polling the Travis API for build status also needs its own token.

**Comment by williamkapke** [comment]
I think ultimately this bot will need many tokens for many services. I'm using a `.env` file and https://www.npmjs.com/package/dotenv to load it.

**Comment by phillipj** [comment]
dotenv is nice. Should we still document which tokens are required and how to generate them in the readme?

**Comment by williamkapke** [comment]
I think an example `.env` file in the readme would do the trick:
```
WEBHOOK_SECRET="hush-hush"
GITHUB_TOKEN="xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
TRAVIS_CI_TOKEN="xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
```


I make use of the `silent: true` option so if there isn't an `.env` file- it doesn't do anything.
```js
require('dotenv').load({ silent: true, path: __dirname + '/.env' })
```

Also, Make sure to add `.env` to the `.gitignore` so there are no accidental commits ;).

File: README.md [hunk 1]
```diff
@@ -2,10 +2,22 @@
...
+
 ## Contributing
 
 Please do, contributions are more than welcome!
 
 ## License
 
-MIT
\ No newline at end of file
+MIT
```
Comment (Fishrock123) [inline README.md]: newline EOF please :)

**Comment by Fishrock123** [comment]
seems good to me

File: README.md (hunk 1 above)
Comment (phillipj) [inline README.md]: > newline EOF please :)

As far as I can see there's a newline EOF present here...

**Comment by phillipj** [comment]
@williamkapke as you introduce dotenv in #8, I'll skip any of that in this PR

**Pull request merged**